src/components/footer/footer.module.scss

```diff
@@ -0,0 +1,46 @@
+.root {
+    padding: 50px 2% 30px;
+    background-color: var(--nude2);
+}
+
+.navigation {
+    display: flex;
+    align-items: start;
+    justify-content: start;
+    gap: 150px;
+    padding: 0 15px 30px;
+}
+
+.navItem {
+    display: block;
+    font: var(--paragraph1);
+    font-family: var(--ff-decor);
+    line-height: 2.2;
+}
+
+.navItem.active {
+    color: var(--brown3);
+}
+
+.bottomBar {
+    display: flex;
+    align-items: baseline;
+}
+
+.logo {
+    flex-grow: 1;
+    font: var(--heading1);
+    line-height: 1;
+    font-size: clamp(120px, 13vw, 220px);
+}
+
+.copyright {
+    flex-shrink: 0;
+    flex-basis: content;
```

Comment:
NIT. `justify-content: space-between` would be enough on the parent instead of flex properties on `logo` and `copyright`. Anyways, you will change their position on smaller screens.